.ci/check_python_dists.sh

```diff
@@ -16,15 +16,15 @@ pip install \
     twine || exit 1
 
 echo "twine check..."
-twine check --strict ${DIST_DIR}/* || exit 1
+twine check --strict "${DIST_DIR}/*" || exit 1
```

Comment:
```suggestion
twine check --strict "${DIST_DIR}/*" || exit 1
```

These are not correct. Putting a wildcard inside quotes will preserve the string with a literal wildcard, instead of expanding it. Test that yourself in a `bash` shell.

```bash
touch out.txt
echo *.txt
# out.txt

echo "*.txt"
# *.txt
```

These instead should be like this:

```bash
"$(echo ${DIST_DIR}/*)"
```

or something like that.

Reply:
It seems you either forgot this or did not understand what I meant. I'll leave inline comments suggesting the specific way these cases should be handled.